### DISCUSSION TITLE
佬们，群成员过多而导致使用get_group_member_list的时候就直接network error：timeout了

### DISCUSSION THREAD

**Discussion by KennyYang0** [howmanybots/onebot#253] [问答] [answered]
<img width="848" alt="截屏2024-02-29 16 11 25" src="https://github.com/botuniverse/onebot/assets/133932142/0da0a99c-2fba-43e4-87a9-e2e6d61e1046">
我已经把API_TIMEOUT设置成了二十分钟，还是不行😭
是不是最好换个设备还是咋整
以及，出了这个意外，还有其他类似可以获取群员qq号的api吗？

**Comment by stdrc** [comment, marked as answer]
OneBot 没有定义 API 分页相关内容，或许可以去找一下你用的 OB 实现看能不能要一下分页接口

**Reply by KennyYang0** [reply]
谢谢佬！虽然但是其实是我用的ls框架的老手机太垃圾了无法完成这个操作，一个50块服务器都能随便完成
